src/main/java/de/zalando/aruha/nakadi/validation/ValidationStrategy.java

```diff
@@ -0,0 +1,26 @@
+package de.zalando.aruha.nakadi.validation;
+
+import java.util.Map;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Maps;
+
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.ValidationStrategyConfiguration;
+
+public abstract class ValidationStrategy {
+
+    private static final Map<String, ValidationStrategy> STRATEGIES = Maps.newHashMap();
+
+    public abstract EventValidator materialize(EventType eventType, ValidationStrategyConfiguration vsc);
+
+    public static final void register(final String name, final ValidationStrategy strategy) {
```

Comment:
This method is static, I don't see much sense to make it final
(the same for the next method)